### DISCUSSION TITLE
[BUG]Issue with New-PnPSite Command

### DISCUSSION THREAD

**Discussion by tatsutanaka** [pnp/powershell#4374] [General]
When I run "New-PnPSite -Type TeamSite -Title $title -Alias $title -Owners $owner", I get the following error: 

New-PnPSite : {"error":{"code":"-1, System.Data.Services.Client.DataServiceRequestException","message":"An error occurred while processing this request."}}

Initially, this command worked perfectly. However, after creating several sites without making any specific changes to the script, this error started to appear. Since this error has started occurring, I have not been able to create any sites.

**Comment by Kal451** [comment]
I'm getting the same from the following

`New-PnPSite -Type TeamSite -TimeZone UTC_GREENWICH_MEAN_TIME_DUBLIN_EDINBURGH_LISBON_LONDON -Title "$New_Site_Title" -Alias $New_Site_URL`

was fine at approx 14:30 GMT+1. came in this morning and while i can create most sites i try (around 30 during testing) 1 or 2 at random produce the error in question.

**Comment by jkaczor** [comment]
Is this a confirmed bug?  Started happening in our environment, and our scripted deployments are blocked.

**Comment by nmsu-adavis** [comment]
I am also running into this issue. I cannot seem to determine what is causing it to happen as I followed all available documentation and verified that none of the information being passed is already in use and that the -owners existed as users in Microsoft.Graph. I believe that this is a bug happening with New-PnPSite. 

Code running:
New-PnPSite -Type TeamSite -Title PnPAutomationTest -Alias PnPAutomationTest -Owners example-email@microsoft.com

Error Recieved:
{"error":{"code":"-1, System.Data.Services.Client.DataServiceRequestException","message":"An error occurred while processing this request."}}

I used Microsoft.Graph to verify the existence of the actual email being passed into owners, and Get-PnPTenantSite to verify that the site being created did not already exist.
